src/pages/index.md

```diff
@@ -13,6 +13,14 @@ keywords:
 
 Experience Platform services provide RESTful APIs that allow you to programmatically perform key operations in your applications.
 
+<AnnouncementBlock slots="heading, text, button" />
+
+### Use the interactive API environment
+
+You can now interact with the Experience Platform API endpoints directly from the API reference pages. Get your [authentication credentials](http://www.adobe.com/go/platform-api-authentication-en) and use the **Try it** functionality in the right rail.
+
+[Learn more](https://experienceleague.adobe.com/en/docs/experience-platform/release-notes/2024/may-2024.md#interactive-api-documentation)
```

Comment:
Before we merge, we'll want to update with the proper live link: https://experienceleague.adobe.com/en/docs/experience-platform/release-notes/2024/may-2024#interactive-api-documentation